apps/remix-ide/src/blockchain/blockchain.js

```diff
@@ -343,14 +346,18 @@ export class Blockchain extends Plugin {
         return this.getProvider() === 'web3' ? this.config.get('settings/personal-mode') : false
       }
     }, _ => this.executionContext.web3(), _ => this.executionContext.currentblockGasLimit())
-
-    this.txRunner = new TxRunner(web3Runner, { runAsync: true })
-    this.txRunner.event.register('transactionBroadcasted', (txhash) => {
+    
+    web3Runner.event.register('transactionBroadcasted', (txhash) => {
       this.executionContext.detectNetwork((error, network) => {
         if (error || !network) return
-        this.event.trigger('transactionBroadcasted', [txhash, network.name])
+        if (network.name === 'VM') return
+        this.call('terminal', 'logHtml',
+          (<a href={etherScanLink(network.name, txhash)} target="_blank">
```

Comment:
Is this link rendered in react? I'm asking because this isn't in a react component?

Reply:
the `logHtml` from the terminal accept either a dom element of a jsx element, so that's not even react.